tests/Functional/bootstrap.php

```diff
@@ -22,7 +22,13 @@
  * our trademarks remain entirely with us.
  */
 
-require __DIR__ . '/../../autoload.php';
+if (file_exists(__DIR__ . '/../../../../../app/autoload.php')) {
+    // shopware as composer dependency
+    require __DIR__ . '/../../../../../app/autoload.php';
+} else {
+    // shopware standalone
+    require __DIR__ . '/autoload.php';
```

Comment:
I believe you wanted to require the same file as before.

```suggestion
    require __DIR__ . '/../../autoload.php';
```

Reply:
Good hint. I had a chat with @soebbing about it and follow a different solution to the autoload file. There is still good solution for the config file missing.